Corona-Warn-App/src/main/java/de/rki/coronawarnapp/nearby/ENFClient.kt

```diff
@@ -0,0 +1,38 @@
+package de.rki.coronawarnapp.nearby
+
+import com.google.android.gms.nearby.exposurenotification.ExposureConfiguration
+import com.google.android.gms.nearby.exposurenotification.ExposureNotificationClient
+import de.rki.coronawarnapp.nearby.modules.diagnosiskeyprovider.DiagnosisKeyProvider
+import timber.log.Timber
+import java.io.File
+import javax.inject.Inject
+import javax.inject.Singleton
+
+@Suppress("DEPRECATION")
+@Singleton
+class ENFClient @Inject constructor(
+    private val googleENFClient: ExposureNotificationClient,
+    private val diagnosisKeyProvider: DiagnosisKeyProvider
+) : DiagnosisKeyProvider {
+
+    // TODO Remove this once we no longer need direct access to the ENF Client,
+    // i.e. in **[InternalExposureNotificationClient]**
+    internal val internalClient: ExposureNotificationClient
+        get() = googleENFClient
+
+    override suspend fun provideDiagnosisKeys(
+        keyFiles: Collection<File>,
+        configuration: ExposureConfiguration?,
+        token: String
+    ): Boolean {
+        Timber.tag(TAG).d(
```

Comment:
The tag isn't necessary in this case IMHO. Timber logs the class name automatically.

Reply:
Generally it's not a KO argument for me to explicitly log a tag, as there are some cases where not specifying an explicit tag will lead automatically generated tags that make filtering the log difficult (some coroutines and callbacks).

I've checked and it's not the case here and will remove the explicit TAG here, thanks!